CHANGES.md

```diff
@@ -2,6 +2,7 @@
 
 Grammars:
 
+- add additional keywords to csp.js (#3244) [Elijah Conners][]
```

Comment:
You need to add this link around line ~17, etc.

Reply:
Done.